src/bokeh/core/_templates/autoload_request_tag.html

```diff
@@ -23,7 +23,7 @@
 .. note::
     This script injects a ``<div>`` in place, so must be placed under ``<body>``.
 
--#}
```

Comment:
@mattpap can you comment on these template changes? I think you are the most invested in the exact formatting of the templates.

Reply:
I made this change to remove the bullet point right above the collapsible block.

![grafik](https://github.com/bokeh/bokeh/assets/68053396/435d0356-51e3-4257-aeae-bb5032189241)

Reply:
Oh that's interesting, but I think it points to an error in our sphinx extension for rendering these templates. Unfortunately the `-` has actual meaning to the template parser (Jinja) which is why I am a bit hesitant to change the template itself. I'd suggest making a dedicated new issue for this so we can follow up and investigate the sphinx extension directly